### ISSUE TITLE
Use ImNodes and ImGuizmo

### ISSUE THREAD

**Issue by erictuvesson** [erictuvesson/CSG#15]
To change the viewer to be more of an editor, it would be nice to use ImNodes and ImGuizmo.
Which was recently merged into the ImGui.NET project.
https://github.com/mellinoe/ImGui.NET/pull/218

Currently waiting on the new projects to be added as NuGet packages.
